Misc Tutorial Improvements, some for noetic (#546)
ros-planning/moveit_tutorials: doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst

RobotState
----------

> The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.

Author: This tutorial comes two after [RobotModel and RobotState](https://ros-planning.github.io/moveit_tutorials/doc/robot_model_and_robot_state/robot_model_and_robot_state_tutorial.html) which makes a distinction between those two pieces, so I updated this bit to match the information from the other tutorial. 

Can you be more specific about what you dislike?

> The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.

Reviewer: Personally I'm not a fan of writing that the `RobotState` examines the geometry of the robot. Saying that it contains a `RobotModel` and joint values would be enough.

> The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.

Author: ```suggestion
The :moveit_core:`RobotState` is a snapshot of a robot. It contains the :moveit_core:`RobotModel` and a set of joint values.
```

> The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.

Reviewer: Good with me at least 👍

> The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.

Author: @AndyZe ?

> The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.

Reviewer 2: Sounds good!
